core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -148,10 +148,6 @@ protected void loggedOut(@Nonnull String username) {
 
     @Extension
     public static final class DescriptorImpl extends UserPropertyDescriptor {
-        public String getDisplayName() {
-            return null;    // not visible
```

Comment:
There are some random finding of mine, too: https://github.com/jenkinsci/ownership-plugin/pull/28#issuecomment-58008037